bundle/manifests/cryostat-operator.clusterserviceversion.yaml

```diff
@@ -54,7 +54,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Monitoring, Developer Tools
     containerImage: quay.io/cryostat/cryostat-operator:2.4.0-dev
-    createdAt: "2023-08-24T00:50:46Z"
+    createdAt: "2023-09-18T15:10:18Z"
```

Comment:
Also need to manually update:

https://github.com/cryostatio/cryostat-operator/blob/893189eacf53652fdb85fdde6cf35f33380a3815/config/manifests/bases/cryostat-operator.clusterserviceversion.yaml#L8

Reply:
If you are interested @aali309, you can handle automation for this step in issue: #494

Reply:
I just tried this out: apparently we can delete the `createdAt` annotation in `config/manifests/bases/cryostat-operator.clusterserviceversion.yaml`, and Operator SDK will generate the annotation when we run `make bundle`. @aali309 Could I ask you to remove the `createdAt` annotation from this file? It should still remain in the generated CSV under `bundle/`.

Reply:
Uhh niceee! That's even better!!

Reply:
@aali309 could you `make bundle` and check in the regenerated files under `bundle/`? This should reset the `createdAt` here (again).

Reply:
I reran it, and the date does get updated.